### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by vanyachyzh**

### CODE REVIEW

File: src/averageAges.js [hunk 1]
```diff
@@ -10,16 +10,25 @@
...
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const diedPeople = people.filter(e => {
+    const deathCentury = Math.ceil(e.died / 100);
+
+    // All men are included if second argument was not passed
+    if (century === deathCentury || century === undefined) {
+      return e.sex === 'm';
```
Comment (vlad-demchuk) [inline src/averageAges.js:23]: no need to leave comments and try to use ternary operator here

Comment (vlad-demchuk) [reply src/averageAges.js:23]: use better variable names instead of `e`

Comment (vlad-demchuk) [reply src/averageAges.js:23]: check all such cases

File: src/averageAges.js [hunk 2]
```diff
@@ -10,16 +10,25 @@
...
+    }
+  });
+
+  const ageSum = diedPeople.reduce((acc, item) => {
+    return acc + (item.died - item.born);
+  }, 0);
+
+  return Number((ageSum / diedPeople.length).toFixed(2));
```
Comment (vlad-demchuk) [inline src/averageAges.js:31]: create a helper function for calculating average value

File: src/averageAges.js [hunk 3]
```diff
@@ -55,7 +79,31 @@ function calculateWomenAverageAge(people, withChildren) {
...
+    let ageDiff = 0;
+
+    // To get index of woman which is mother of someone
+    if (onlyWithSon) {
+      indexMother = people.findIndex(mother => {
+        return mother.name === child.mother && child.sex === 'm';
+      });
+    } else {
+      indexMother = people.findIndex(mother => mother.name === child.mother);
+    }
```
Comment (vlad-demchuk) [inline src/averageAges.js:95]: the same, it would be better to use ternary operator

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
remove comments. your code should be clear with your variable names.

**Review by Anastasiia-Svintsova — APPROVED** [review]

**Pull request closed without merging**